ansi/ansi.go

```diff
@@ -1,10 +1,8 @@
 package ansi
 
-import "regexp"
-
-var ansiEscape = regexp.MustCompile(`\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])`)
+import "github.com/charmbracelet/x/ansi"
 
 // Strip strips a string of any of it's ansi sequences.
 func Strip(text string) string {
```

Comment:
If you have only this in the package, replace all call to ansi.Strip by the x/ansi one.

Or at least add a 


```suggestion
// Deprecated: use x/ansi
 func Strip(text string) string {
```

Reply:
Yes, I was thinking about dropping `gum/ansi` altogether, just not sure about the impact

CC/ @maaslalani